src/ape_pm/compiler.py

```diff
@@ -78,3 +80,62 @@ def compile(
             contract_types.append(contract_type)
 
         return contract_types
+
+    def enrich_error(self, err: ContractLogicError) -> ContractLogicError:
```

Comment:
one thing is that this is exactly the same implementation `ape-solidity` would have... wondering if we should use a shared method somehow?

Reply:
- [ ] going to make ecosystem decode_error method, that should help